### ISSUE TITLE
Twitch and YouTube Support

### ISSUE THREAD

**Issue by airpioa** [t41372/Open-LLM-VTuber#59]
Adding support for open llm v-tuber to talk with twitch and youtube chat.

**Comment by t41372** [comment]
Good idea. Integrating Twitch, YouTube, and Bilibili chat are important features, considering the word "vtuber" is used in the name of this project.

However, this project is undergoing massive refactoring, and I probably won't have time to do this feature in the coming weeks. This feature will ultimately be added once the refactoring is done.

In the meantime, any PRs are welcome (please, based on the `superb-refactoring` branch if anyone is interested. It's the refactoring branch that will become `v1.0.0`, which is the next version).
